client/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+    <head>
+        <title>Client Served</title>
```

Comment:
I think you ideally want this to be the title of your site as it will show up in the browser tab

Reply:
yep, i'll replace the index.html with my correct one when i get home tonight